CHANGELOG-1.2.md

```diff
@@ -0,0 +1,10 @@
+# Changelog since v1.1.0
+
+## Breaking Changes
+
+- Changes the API group name for the fake VolumeSnapshot object to "snapshot.storage.k8s.io" to be in-sync with the group name of the real VolumeSnapshot object. As a result, the generated interfaces for clientset and informers of VolumeSnapshot are also changed from "VolumeSnapshot" to "Snapshot". ([#123](https://github.com/kubernetes-csi/external-snapshotter/pull/123), [@xing-yang](https://github.com/xing-yang))
+
+
+## Other Notable Changes
```

Comment:
do you also want to mention the finalizer changes?

Reply:
Good catch!  Yes, added that.  It was missed because the Finalizer PR was submitted longer time ago before we had the PR description template.  The release notes tool only picks up the PRs with a release note.